### PR TITLE
pythonPackages.gusmobile: init at unstable-2020-03-17

### PR DESCRIPTION
**Pull request by jb55**
```
A gemini protocol python library. This is needed for the CAPCOM gemini
feed aggregator.

Signed-off-by: William Casarin <jb55@jb55.com>
Reviewed-by: Matthias Beyer <mail@beyermatthias.de>
Reviewed-by: Xinglu Chen <public@yoctocell.xyz>
Message-Id: 20210206005608.10821-1-jb55@jb55.com
Link: https://lists.sr.ht/~andir/nixpkgs-dev/patches/20061
```

### CODE REVIEW

File: pkgs/development/python-modules/gusmobile/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  };
+
+  meta = with lib; {
+    homepage = "https://git.carcosa.net/jmcbray/gusmobile";
+    description = "A simple library for making Gemini protocol requests";
+    license = licenses.agpl3;
+    maintainers = [ jb55 ];
+  };
+
+}
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/gusmobile/default.nix:20]:
```suggestion
  };
}
```

File: pkgs/development/python-modules/gusmobile/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  src = fetchgit {
+      url = "https://git.carcosa.net/jmcbray/gusmobile.git";
+      rev = "0a09aaaf00578d1389a4aa6b15f5c64867e0f8ca";
+      sha256 = "0wpvfk5kj23r83xfha6kidsh5c7y143y9ximlx3h92hih7rpskmn";
+  };
+
+  meta = with lib; {
+    homepage = "https://git.carcosa.net/jmcbray/gusmobile";
+    description = "A simple library for making Gemini protocol requests";
+    license = licenses.agpl3;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/gusmobile/default.nix:16]: agpl3Plus or agpl3Only?

File: pkgs/development/python-modules/gusmobile/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    maintainers = [ jb55 ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/gusmobile/default.nix:17]: Please add yourself as a maintainer in a separate commit with the message ``maintainers: add jb55``.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by jb55** [comment]
Thanks!

> agpl3Plus or agpl3Only?

not sure, I'll double check

> +    maintainers = [ jb55 ];
>
> Please add yourself as a maintainer in a separate commit with the message ``maintainers: add jb55``.

I am already a maintainer but it looks like I forgot to reference
`maintainers.jb55` here. I always do this... annoying that there's no
evaluation error at build time.

File: pkgs/development/python-modules/gusmobile/default.nix [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+buildPythonPackage {
+  pname = "gusmobile";
+  version = "unstable-2020-03-17";
+
+  src = fetchgit {
+      url = "https://git.carcosa.net/jmcbray/gusmobile.git";
+      rev = "0a09aaaf00578d1389a4aa6b15f5c64867e0f8ca";
+      sha256 = "0wpvfk5kj23r83xfha6kidsh5c7y143y9ximlx3h92hih7rpskmn";
+  };
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/gusmobile/default.nix:11]:
```suggestion
    url = "https://git.carcosa.net/jmcbray/gusmobile.git";
    rev = "0a09aaaf00578d1389a4aa6b15f5c64867e0f8ca";
    sha256 = "0wpvfk5kj23r83xfha6kidsh5c7y143y9ximlx3h92hih7rpskmn";
  };
```

**Comment by SuperSandro2000** [comment]
> I am already a maintainer but it looks like I forgot to reference `maintainers.jb55` here. I always do this... annoying that there's no evaluation error at build time.

You need to turn on ``checkMeta = true;`` in ~/.config/nixpkgs/config.nix.
Also you can write the full length of a line.

**Comment by jb55** [comment]
Sandro <notifications@github.com> writes:
> You need to turn on ``checkMeta = true;`` in ~/.config/nixpkgs/config.nix.

oh very nice. thanks.

pushed a new revision with fixed spaces.

**Pull request closed without merging**